packages/apollo-server-core/src/ApolloServer.ts

```diff
@@ -129,16 +130,41 @@ export class ApolloServerBase {
     this.requestOptions = requestOptions as GraphQLOptions;
     this.context = context;
 
+    if (uploads !== false) {
+      if (this.supportsUploads()) {
+        if (uploads === true || typeof uploads === 'undefined') {
+          this.uploadsConfig = {};
+        } else {
+          this.uploadsConfig = uploads;
+        }
+        //This is here to check if uploads is requested without support. By
+        //default we enable them if supported by the integration
+      } else if (uploads) {
+        throw new Error(
+          'This implementation of ApolloServer does not support file uploads.',
```

Comment:
Is there any way we can make this more specific? We want to make sure people understand it is because of the framework/environment they are using.